### ISSUE TITLE
Rendered merge conflict on the security cookbook.

### ISSUE THREAD

**Issue by matt-usurp** [symfony/symfony-docs#5722]
On the below page there is a block of code that appears to have been rendered from a merge conflict.

http://symfony.com/doc/current/cookbook/security/voters.html

```
<<<<<<< HEAD
$this->denyAccessUnlessGranted('view', $post, 'Unauthorized access!');
throw new AccessDeniedException('Unauthorized access!');
}
>>>>>>> 2.3

return new Response('<h1>'.$post->getName().'</h1>');
}
}
```

Sorry I am not able to submit a merge request to fix this at this time but also I don't feel like I know enough about the security components (hence me reading up on it) to be able to resolve this issue.

**Comment by jzawadzki** [comment]
already fixed in 713541a5151334aaab1d48892ba6803aac6c3add

**Comment by xabbuh** [comment]
You are right. I made that mistake when merging and unfortunately noticed it too late (the documentation was already generated again). The good news is that the docs have been built again in the meantime and the issue is gone. Anyway, thank very much for reporting this @matt-usurp.
